admin/src/main/scala/io/buoyant/admin/AdminConfig.scala

```diff
@@ -9,13 +10,14 @@ case class AdminConfig(
   port: Option[Port] = None,
   shutdownGraceMs: Option[Int] = None,
   tls: Option[TlsServerConfig] = None,
-  httpIdentifierPort: Option[Port] = None
+  httpIdentifierPort: Option[Port] = None,
+  workerThreads: Option[Int] = None
 ) {
 
-  def mk(defaultAddr: InetSocketAddress): Admin = {
+  def mk(defaultAddr: InetSocketAddress, stats: StatsReceiver): Admin = {
     val adminIp = ip.getOrElse(defaultAddr.getAddress)
     val adminPort = port.map(_.port).getOrElse(defaultAddr.getPort)
     val addr = new InetSocketAddress(adminIp, adminPort)
-    new Admin(addr, tls)
+    new Admin(addr, tls, workerThreads.getOrElse(2), stats)
```

Comment:
Does this need to be validated? It seems that we only should really support values on ~ [1-64]